sdk-base/src/main/java/com/mapbox/maps/plugin/attribution/AttributionView.kt

```diff
@@ -42,5 +42,5 @@ interface AttributionView {
   /**
    * Set an [View.OnClickListener] to AttributionView
    */
-  fun setOnClickListener(listener: View.OnClickListener)
+  fun setOnViewClickListener(listener: View.OnClickListener)
```

Comment:
Why are we duplicating word view: `AttributionView.setOnViewClickListener`?
I'm not really against it but just curious.

Reply:
It complained about an accidental overrride. Interface method with the same name as inherited from View.